e2e/steps_file.js

```diff
@@ -194,6 +202,16 @@ module.exports = function () {
       await this.completeEvent('Save and continue');
     },
 
+    async fillDate(date, dateId){
+      if(date) {
+        return within(dateId ? dateId : 'form', () => {
```

Comment:
Would this be better?
```suggestion
    async fillDate(date, dateId = 'form'){
      if(date) {
        return within(dateId, () => {
```

Reply:
you are right, default value makes sense, thanks